app/Models/Server.php

```
@@ -124,10 +127,45 @@ class Server extends AbstractModel
 
     public bool $deleteFromProvider = true;
 
+    /** @var array<int, array<int, int>> */
+    protected static array $networkSiblingsToResync = [];
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $networkSiblingsToResync. Keep variable name length under 20. (undefined)

(LongVariable)